README.md

```diff
@@ -7,25 +7,45 @@
 
 ## Group Members
 
+1. Brent
+2. Coulter
+3. Jackson
+
 
 ## Project Description
 
+Building a web-app for users to connect with developers and pair program
+
```

Comment:
This doesn't give a complete picture of what you're building. Expand on what exactly you're building (an app that allows screen sharing and live chat, etc)